app/views/users/edit_website_settings.html.erb

```diff
@@ -0,0 +1,20 @@
+<% title "Edit Website Settings: #{@user.name}" %>
+
+<%= link_to @user.name, @user %> → Edit Login credentials
+<h1>Edit Website Settings</h1>
+
+
+<%= form_for @user do |f| %>
+  <table>
+    <tbody>
+      <tr>
+        <td>Show exact UTC times for posts and comments</td>
+        <td>
+          <%= f.select :utc_time, [["No", false], ["Yes", true]], {}, { disabled: !can_edit? } %>
```

Comment:
Please change this to a checkbox, since it is only 2 different states the option can be (true or false)